tools/webdriver/webdriver/bidi/modules/script.py

```diff
@@ -0,0 +1,53 @@
+from typing import Any, Optional, Mapping, MutableMapping, Union, Dict
+from enum import Enum
+
+from ._module import BidiModule, command
+
+
+class ScriptEvaluateResultException(Exception):
+    def __init__(self, result: Mapping[str, Any]):
+        self.result = result
+        super().__init__("Script execution failed.")
```

Comment:
`BidiException` represents BiDi error message. [`ScriptEvaluateResultException` is a valid response instead](https://w3c.github.io/webdriver-bidi/#type-script-ScriptEvaluateResult), so I wouldn't merge them.

Reply:
Ok, I see. When reading `The ExceptionDetails type represents a JavaScript exception.` in the BiDi spec I thought that for clients it might be good to bind it to this exception type, but it looks like we shouldn't and a JavaScript error is now differently reported as for WebDriver classic.